src/main/java/ee/tuleva/onboarding/aml/AmlService.java

```diff
@@ -0,0 +1,65 @@
+package ee.tuleva.onboarding.aml;
+
+import ee.tuleva.onboarding.auth.principal.Person;
+import ee.tuleva.onboarding.epis.EpisService;
+import ee.tuleva.onboarding.epis.contact.UserPreferences;
+import ee.tuleva.onboarding.user.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Example;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AmlService {
+
+    private final AmlCheckRepository amlCheckRepository;
+    private final EpisService episService;
+
+    public void checkUserAfterLogin(User user, Person person) {
+        addCheckIfMissing(user, AmlCheckType.DOCUMENT, true);
```

Comment:
Remove boolean variable

Reply:
A better way could be to pass in an enum for the AML check state (i.e. SUCCESS, FAIL) since it has better readability than passing in "true" as an argument.

Reply:
true :trollface: